src/widget/core.rs

```diff
@@ -28,6 +37,7 @@ pub enum Event {
 }
 
 /// Static state that is shared between most contexts.
+#[derive(Debug)]
```

Comment:
Not sure where this is necessary but with it, it doesn't compile since `Message` needs to be `Debug` (and I'm not sure whether we want this to be `Debug` anyway)

Reply:
I thought about https://rust-lang.github.io/api-guidelines/interoperability.html#c-common-traits and because I wanted it for debugging myself I added it and left it.
If you don't intend to go in that direction I'll just remove it and also stay away from adding it to others.

Reply:
I couldn't reproduce the build issue, but I'll just remove it.